sdc/datatypes/hpat_pandas_series_functions.py

```diff
@@ -3465,7 +3465,7 @@ def hpat_pandas_series_rename(self, index=None, copy=True, inplace=False, level=
 
     Limitations
     -----------
-    - Parameter level is currently unsupported by Intel Scalable Dataframe Compiler
+    - Parameter ``level`` is currently unsupported by Intel Scalable Dataframe Compiler
```

Comment:
May be a bit redundant? It's ok in short variant I think
```suggestion
    - Parameter ``level`` is currently unsupported.
```